### DISCUSSION TITLE
Updating Content Type property EditFormClientSideComponentId does not work

### DISCUSSION THREAD

**Discussion by EA12** [pnp/powershell#2158] [Q&A] [answered]
The value will not be persisted.
SharePoint-Online, modern page.
I am logged in as SharePoint Admin and Global Admin.
I need this, because I delevoped a SPFx-Solution (Form Customizer) and must associate the Extension with the list content type.
Am I doing something wrong?

The code I used:

```Import-Module PnP.PowerShell
$siteURL="https://contoso.sharepoint.com/sites/xxx"
Connect-PnPOnline -Url $siteURL -Interactive
$ContentTypeName="ZZZ"
$CTypes = Get-PnPContentType -List "List2"
foreach($contentType in $CTypes)
{
	if($contentType.Name -eq $ContentTypeName)
	{
		$contentType.DisplayFormClientSideComponentId = "50ebd4b2-04f4-42fb-af43-953e226179f0";
        $contentType.NewFormClientSideComponentId = "50ebd4b2-04f4-42fb-af43-953e226179f0";
        $contentType.EditFormClientSideComponentId = "50ebd4b2-04f4-42fb-af43-953e226179f0";
		$contentType.Update($false)
	}
}```

**Comment by React365-Chris** [comment]
I also have the same concern. There is no documentation on how to accomplish this. If I find it I will reply to you here.

**Comment by React365-Chris** [comment, marked as answer]
This worked for me. Good Luck!

#Login
Connect-PnPOnline -Url http://MyServer/sites/MySiteCollection -Credentials $creds
 
# Get Context
$clientContext = Get-PnPContext
 
# Give target content type name over here
$targetContentType = Get-PnPContentType -Identity "Content Type Title" -List "Your List"
 
# Set target content type read only
$targetContentType.EditFormClientSideComponentId = '8edc4f83-3e73-4d31-9828-b67b7b786667'
 
# Update(UpdateChildren – bool), this value indicates whether the children content type(inheriting from this Content Type) needs to be updated. 0 = False, 1 = True
$targetContentType.Update(0)
 
$clientContext.ExecuteQuery()

**Reply by EA12** [reply]
Thanks Marc, this did the trick

**Reply by carletto75** [reply]
$clientContext worked for me. I had same issue. Thanks
